example.docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+Version: '3'
+Service:
+  Pufferpanel:
+    Image: pufferpanel/pufferpanel
+    Container_name: Pufferpanel
+    Restart: on-failure
+    Environment: 
+      - 'PUFFER_LOGS=/etc/pufferpanel/logs', # Location of your logs
+      - 'PUFFER_PANEL_DATABASE_DIALECT=sqlite3', # Databse type https://docs.pufferpanel.com/en/2.x/guides/database.html
+      - 'PUFFER_PANEL_DATABASE_URL=file:/etc/pufferpanel/pufferpanel.db?cache=shared', # SQL URL https://docs.pufferpanel.com/en/2.x/guides/da>
+      - 'PUFFER_DAEMON_DATA_CACHE=/var/lib/pufferpanel/cache', # Cache location within the container
+      - 'PUFFER_DAEMON_DATA_SERVERS=/var/lib/pufferpanel/servers', # Server data location within the container
+      - 'PUFFER_DAEMON_DATA_MODULES=/var/lib/pufferpanel/modules', # Location for Modules within the container
+
+    Volumes:
+      - '/etc/pufferpanel:/etc/pufferpanel', # Config/logs folder
```

Comment:
Use a volume for the config directory instead of the host.

Reply:
Documentation sake. Avoids maintaining separate version of docs for depending on installation method.

Reply:
We already use docker volumes for the config when plain docker is used. This should essentially match those steps.

Reply:
switched to volumes.